test/sharness/t0020-init.sh

```diff
@@ -81,9 +81,9 @@ test_expect_success "ipfs peer id looks good" '
 
 test_expect_success "ipfs init output looks good" '
   STARTFILE="ipfs cat /ipfs/$HASH_WELCOME_DOCS/readme" &&
-  echo "initializing IPFS node at $IPFS_PATH" >expected &&
-  echo "generating $BITS-bit RSA keypair...done" >>expected &&
+  echo "generating ED25519 keypair...done" >expected &&
   echo "peer identity: $PEERID" >>expected &&
+  echo "initializing IPFS node at $IPFS_PATH" >>expected &&
```

Comment:
The init tests don't take any flags which is why these tests are going to fail now (we switched back to RSA defaults without changing the tests).

We'll probably need to do a copy-paste here so we can handle Ed25519, RSA, and none. What do you think?

Reply:
See if the newly pushed code looks flexible enough.